server/data/data-api/src/test/java/org/apache/james/rrt/lib/MappingTest.java

```diff
@@ -54,6 +54,12 @@ void hasPrefixShouldReturnTrueWhenForward() {
         assertThat(hasPrefix).isTrue();
     }
 
+    @Test
+    void hasPrefixShouldReturnTrueWhenAlias() {
+        boolean hasPrefix = Mapping.Type.hasPrefix(Type.Alias.asPrefix() + "myRegex");
```

Comment:
Shouldn't it be `myAlias`?

Reply:
Seems to be `myRegex` for all the other tests as well. So I will fix it in an other commit like in your other comments